frontend/src/App.tsx

```diff
@@ -1,24 +1,37 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+// react dom
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+    <div className="App" /* later will be in other file, export const Login = () => { */>
+      <header className="LoginSignUp-header">
```

Comment:
We should implement a router and move the Login page into its own component (Like in one of the lab). Also for this PR i think we should write some basic tests.